student.rb

```diff
@@ -1,16 +1,19 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
```

Comment:
- [ ] When setting the classroom for a student it also adds it but just as a `attr_reader` not as attr_accessor as required

![image](https://user-images.githubusercontent.com/58553711/194444938-290b9da4-d23d-4553-acc5-fc8ea9c252ca.png)